.github/workflows/daily-tests.yaml

```diff
@@ -363,9 +365,189 @@ jobs:
         name: ${{needs.build-gem5-ALL_MESI_Two_Level.outputs.build-name}}
         path: build/ALL_MESI_Two_Level
     - run: chmod u+x build/ALL_MESI_Two_Level/gem5.opt
-    - name: long gem5_library_example_tests
+    - name: long gem5_library_example_tests_x86_ubuntu_run_ALL_x86_64_opt
       working-directory: ${{ github.workspace }}/tests
-      run: ./main.py run gem5/gem5_library_example_tests --length=long --skip-build -vv -t $(nproc)
+      run: ./main.py run --uid SuiteUID:tests/gem5/gem5_library_example_tests/test_gem5_library_examples.py:test-gem5-library-example-x86-ubuntu-run-ALL-x86_64-opt --length=long --skip-build -vv
+    - name: create zip of results
+      if: success() || failure()
+      run: |
+        apt-get -y install zip
+        zip -r output.zip tests/testing-results
+    - name: upload zip
+      if: success() || failure()
+      uses: actions/upload-artifact@v3
+      env:
+        MY_STEP_VAR: ${{github.job}}_COMMIT.${{github.sha}}_RUN.${{github.run_id}}_ATTEMPT.${{github.run_attempt}}
```

Comment:
Does this say "gem5_library_example_tests_x86_ubuntu_run_ALL_x86_64_opt" somewhere in the filename?

Reply:
Yeah, ${{github.job}} does that, so we don't have duplicate names in the artifact output.